dissect/target/plugins/filesystem/ntfs/mft.py

```diff
@@ -90,6 +90,21 @@
     ],
 )
 
+FilesystemMCABRecord = TargetRecordDescriptor(
+    "filesystem/ntfs/mft/mcab",
```

Comment:
```suggestion
    "filesystem/ntfs/mft/macb",
```
And everywhere else.